contrib/hbcrypto/hbcrypto

```diff
@@ -0,0 +1,46 @@
+/* --------------------------------------------------------------------
```

Comment:
Is not this file a copy of hbcrypto.hbx? Is it needed here?

Reply:
Certainly not. Thanks for spotting. Probably was created by wrong .hbx specifier in .hbp file - a difference between Harbour 3.2 and 3.4 hbmk2.